besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -554,6 +554,29 @@ void setBannedNodeIds(final List<String> values) {
       description = "Set if node will perform mining (default: ${DEFAULT-VALUE})")
   private final Boolean isMiningEnabled = false;
 
+  @Option(
+      names = {"--cpu-miner-enabled"},
+      description = "Set if node will perform CPU mining (default: ${DEFAULT-VALUE})")
+  private final Boolean isCpuMiningEnabled = false;
+
+  @SuppressWarnings("FieldMayBeFinal") // Because PicoCLI requires Strings to not be final.
+  @Option(
+      names = {"--stratum-host"},
```

Comment:
to align with the other cli, make this `--mining-stratum-host` along with backing variable.  Same with port and extranonce.

Reply:
ok this is done now.